### DISCUSSION TITLE
tine podcast: 24 – Responsive Design

### DISCUSSION THREAD

**Discussion by pschuele** [tine-groupware/tine#78] [Announcements]
Herzlich willkommen zum tine Podcast! Wir möchten Euch zu einem Blick hinter die Kulissen der Open-Source-Groupware tine mitnehmen. Achtung! Manchmal kann es technisch werden.

Das Design von tine ruft immer wieder Diskussionen hervor: Auf manche wirkt es altbacken, man könnte es auch „Retro“ nennen, aber Änderungen sind erstens Geschmackssache und zweitens in einem komplexen System wie tine nur mit einem wohldurchdachten Plan machbar. Cornelius Weiß berichtet über anstehende Modernisierungen und die einhergehenden Herausforderungen.

Gebt uns gerne Feedback zu dieser Episode. Habt Ihr vielleicht Themen, über die wir unbedingt mal sprechen sollten oder andere Anregungen? [Her damit!](https://www.tine-groupware.de/kontakt/) Wenn es Euch gefallen hat, hinterlasst uns doch eine positive Bewertung im Podcast-Portal Eurer Wahl.

Neue Episoden erscheinen immer am vierten Mittwoch des Monats – am besten richtet Ihr Euch schon mal einen Serientermin in tine ein.

https://www.tine-groupware.de/podcast/24-responsive-design/

**Comment by ignoreigor** [comment]
Hi,
finde die Designänderung eigentlich ganz passend.  Wird für mich nutzbarer, wenn ich tatsächlich die GUI mal mobil nutze. Application-Chooser und Icons links braucht Gewöhnung, aber ist kein Problem. Einige kompaktierte Responsive-Listen sind zusammengefasst noch nicht sprechend, weil angezeigte Infos im eingeklappten Zustand sehr knapp ist.  Oberfläche altbacken? Vielleicht... aber sehr funktionell und übersichtlich. Das sollte trotz einer optischen Modernisierung nicht verloren gehen.

Was mich umtreibt: Wo ist "Über Tine", um die Version der UI zu checken? Dann noch eine Frage: Der Fenstertyp "Overlay-Windows" sollte doch mal raus. Der ist wieder da, funktioniert gefühlt schneller, als der Standard "native Windows", die jedoch zuverlässiger sind.

**Reply by pschuele** [reply]
hallo,
danke für dein feedback. das mit "Über tine" habe ich weitergegeben bzw. zur diskussion gestellt (vielleicht sollte es oben rechts wieder ein tine icon geben, wo diese sachen abgerufen werden können?). finde auch, dass man diese infos nicht einfach über bord werfen sollte ...

**Reply by pschuele** [reply]
> Was mich umtreibt: Wo ist "Über Tine", um die Version der UI zu checken?

ist jetzt wieder drin: b953e8daeea56db00cedad5bcb575b0c56ad29a1 

![Screenshot 2024-10-10 at 09 32 04](https://github.com/user-attachments/assets/7d828e20-2be1-4df2-880d-b2a097629773)

**Reply by ignoreigor** [reply]
Daumen hoch. Ein bisschen mehr tine-Branding konnte dem UI auch nicht schaden ;-)
